web_ir_actions_act_window_page/README.md

```diff
@@ -0,0 +1,12 @@
+Client side paging
+=====================
+
+This addon enables buttons to return::
```

Comment:
Yeah, but you use it as a return value for the method of a button to perform a client side action